src/cartridges/int_adyen_SFRA/cartridge/client/default/js/applePayExpress.js

```diff
@@ -125,8 +121,9 @@ function callPaymentFromComponent(data, resolveApplePay, rejectApplePay) {
 
 initializeCheckout()
   .then(async () => {
+    const paymentMethodsResponse = await getPaymentMethods();
     const applePayPaymentMethod =
-      checkout.paymentMethodsResponse.paymentMethods.find(
+      paymentMethodsResponse.AdyenPaymentMethods.find(
```

Comment:
here its good to check whether paymentMethodsResponse has value

`paymentMethodsResponse?.AdyenPaymentMethods.find()`